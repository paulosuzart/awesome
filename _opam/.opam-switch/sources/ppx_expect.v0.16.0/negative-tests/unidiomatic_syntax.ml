[%%expect_test
  let _ =
    Printf.printf "Hello, world.\n";
    [%expect {| Good night, moon. |}]]
;;
