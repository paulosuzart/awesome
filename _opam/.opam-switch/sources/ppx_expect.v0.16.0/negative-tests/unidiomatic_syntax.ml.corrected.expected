[%%expect_test
  let _ =
    Printf.printf "Hello, world.\n";
    [%expect {| Hello, world. |}]]
;;
